.github/workflows/cd.yml

```
@@ -0,0 +1,121 @@
+name: 'Spendr Continuous Deployment'
+
+on:
+  push:
+    branches: ['ci-cd-configurations']
+  pull_request:
+    branches: ['ci-cd-configurations']
+
+jobs:
+  build-server:
+    name: 'spendr server continuous deployment build'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
```

Nit: For consistency with other jobs

```suggestion
    steps:
    - name: Check out code
    - uses: actions/checkout@v3
```